pytometry/read_write/_readfcs.py

```diff
@@ -16,3 +22,51 @@ def read_fcs(path: str, reindex: bool = True) -> AnnData:
         an AnnData object of the fcs file
     """
     return readfcs.read(path, reindex=reindex)
+
+
+def read_and_merge(
+    files: str | list[str],
+    sample_ids: list[Any] | None = None,
+    sample_id_from_filename: bool = False,
+    sample_id_index: int = 0,
+    sample_id_sep: str = "_",
+) -> AnnData:
+    """Read and merge multiple FCS files into a single AnnData object with an
```

Comment:
Lint error on the formatting of the docstring. Please run `pre-commit` on all files for a quick turnaround.

Reply:
Hmm...I was running pre-commit on all my commits. I've never used flit before (I tend to use poetry) so maybe I set something up wrong? I'll double check my pre-commit config